pytest-pixl/tests/test_ftpserver_fixture.py

```diff
@@ -11,11 +11,32 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
+import os
+from pathlib import Path
+
 import pytest
+from conftest import TEST_DIR
+from pytest_pixl.dicom import _create_default_json
+from pytest_pixl.plugin import FtpHostAddress
 
 
-@pytest.mark.pytester_example_path("tests/samples_for_fixture_tests/test_ftpserver_fixture")
+@pytest.mark.pytester_example_path(
+    str(TEST_DIR) + "/samples_for_fixture_tests/test_ftpserver_fixture"
```

Comment:
FYI this is a bit cleaner IMHO
```suggestion
    f"{TEST_DIR}/samples_for_fixture_tests/test_ftpserver_fixture"
```

Reply:
Thanks, sorted out here [c3e4e5f](https://github.com/SAFEHR-data/PIXL/pull/501/commits/c3e4e5fd68a19b1ef3c2303005ed097c73b35f03)